packages/react-positioning/src/useMouseTarget.ts

```diff
@@ -10,8 +10,8 @@ import * as PopperJs from '@popperjs/core';
  * @param initialState - initializes a user provided state similare to useState
  * @returns state and dispatcher for a Popper virtual element that uses native/synthetic mouse events
  */
-export const usePopperMouseTarget = (initialState?: PopperJs.VirtualElement | (() => PopperJs.VirtualElement)) => {
-  const [virtualElement, setVirtualElement] = React.useState<PopperJs.VirtualElement | undefined>(initialState);
+export const useMouseTarget = (initialState?: PositioningVirtualElement | (() => PositioningVirtualElement)) => {
```

Comment:
```suggestion
export const usePositioningMouseTarget = (initialState?: PositioningVirtualElement | (() => PositioningVirtualElement)) => {
```

Can we keep "positioning" in the name? Without reference to positioning it looks confusing TBH.